### PR TITLE
WebTestCase: preFixtureRestore() method

### PR DESCRIPTION
**Pull request by kniziol**
Executed **before** restore fixtures. Required to do something before load the fixtures.

### CODE REVIEW

**Comment by lsmith77** [comment]
fine for me .. wondering if we should also add those parameters to the "post" method?

**Comment by kniziol** [comment]
@lsmith77 It's good suggestionsə, I think

**Comment by lsmith77** [comment]
can you add it to the PR?

**Comment by kniziol** [comment]
@lsmith77 Yes. It's done.

**Comment by kniziol** [comment]
I have created ```preReferenceSave()``` and ```postReferenceSave()``` methods

File: Test/WebTestCase.php [hunk 1]
```diff
@@ -346,9 +353,51 @@ protected function postFixtureSetup()
...
+
+    /**
+     * Callback function to be executed before save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
      */
-    protected function postFixtureRestore()
```
Comment (lsmith77) [inline Test/WebTestCase.php]: actually that was a stupid suggestion by me .. because it will break BC, so can you please remove the new parameters to ``postFixtureRestore()``?

Comment (kniziol) [inline Test/WebTestCase.php]: OKey, I will remove those arguments, but... it will be good to access instance of the ```ObjectManager``` or ```ProxyReferenceRepository``` directly in ```postFixtureRestore()``` method. What do you think about it?

Comment (kniziol) [inline Test/WebTestCase.php]: @lsmith77 are you there? :]

Comment (lsmith77) [inline Test/WebTestCase.php]: sorry .. I will be off for 10 days and work as been getting ever so more stressful in the lead up time. so open source work kind of got the short end of the stick. I hope to wrap this up in the first days once I am back on the 15th.

Comment (lsmith77) [inline Test/WebTestCase.php]: ok I am back.

yeah it would be good but I would like to avoid BC breaks unless we do a bigger refactoring. Now I am open to such a bigger refactoring (see https://github.com/liip/LiipFunctionalTestBundle/pull/136/files) in case someone has time to finish it up. Until then it would be better to revert.

Comment (kniziol) [inline Test/WebTestCase.php]: I understand, but when refactoring will be finished?

Comment (lsmith77) [inline Test/WebTestCase.php]: I do not know, as you may have noticed this Bundle is not necessarily on the top of my agenda right now. The above PR is already quite far along. So if you have some time to work on it .. it could likely get merged soon.

Comment (kniziol) [inline Test/WebTestCase.php]: I don't have time :]
So, what finally do with arguments of the ```postFixtureRestore()``` method?

Comment (lsmith77) [inline Test/WebTestCase.php]: keep them as they were before this PR, ie. revert the change in this PR

Comment (kniziol) [inline Test/WebTestCase.php]: OKay. It's done.
